internal/osbuild/assembler_test.go

```diff
@@ -0,0 +1,164 @@
+package osbuild
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/google/uuid"
+	"reflect"
+	"testing"
+)
+
+func TestAssembler_UnmarshalJSON(t *testing.T) {
+	//nullUUID := uuid.MustParse("00000000-0000-0000-0000-000000000000")
+	type fields struct {
+		Name    string
+		Options AssemblerOptions
+	}
```

Comment:
this looks like `type Assembler struct` from assembler.go. 

Why does the test have to define its own data type? 
Can't it just reuse the `Assembler` struct instead ?